docker/Dockerfile.frontend

```diff
@@ -1,7 +1,9 @@
 FROM node:16.13.0 AS frontend
 
+ENV YARN_CACHE_FOLDER /home/node/.yarn-cache
```

Comment:
*Ditto

Reply:
@shivang0-0 If we are changing yarn-cache location, don't we need to change it in Makefile/lin95. Also why are we changing yarn location?
cc @gp201